app/src/main/java/com/likeminds/feedsx/FeedSXApplication.kt

```diff
@@ -1,7 +1,22 @@
 package com.likeminds.feedsx
 
 import android.app.Application
+import com.likeminds.likemindsfeed.LMFeedClient
+import com.likeminds.likemindsfeed.sdk.model.InitiateLikeMindsExtra
 import dagger.hilt.android.HiltAndroidApp
 
 @HiltAndroidApp
-class FeedSXApplication : Application()
\ No newline at end of file
+class FeedSXApplication : Application() {
+    companion object {
+        const val LOG_TAG = "LikeMinds"
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        // extras to instantiate LMFeedClient
+        val extra = InitiateLikeMindsExtra.Builder()
+            .application(this)
+            .build()
```

Comment:
remove this class from `data-layer` and add in Builder pattern of `LMFeedClient`